Makefile

```diff
@@ -286,15 +304,13 @@ $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATF
 assets/osx/installer.tiff: assets/osx/installer.png assets/osx/installer@2x.png
 	tiffutil -cathidpicheck $^ -out $@
 
-$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-darwin-$(TARGET_ARCH).dmg: assets/osx/installer.tiff \
-	| $(BUILD_DIRECTORY)
+electron-installer-dmg: assets/osx/installer.tiff
```

Comment:
I'm :-1: on this change (and the similar changes) - I think it's useful for the `Makefile` to still refer to the actual filenames that get generated, rather than only using phoney targets everywhere.